service/shipstation.rest.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- General Guideline Verbs: GET=find, POST=create, PUT=store (create or update), PATCH=update, DELETE=delete  -->
+
+<resource xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://moqui.org/xsd/rest-api-2.1.xsd" name="shipstation" displayName="ShipStation API" version="1.0.0" description="API for ShipStation Integration">
+
+    <resource name="orders" require-authentication="anonymous-all">
+        <method type="get"><service name="mantle.shipstation.ShipStationServices.set#GetCall"/></method>
+    </resource>
+    <resource name="data" require-authentication="anonymous-all">
```

Comment:
Please check, there must be some kind of validation to get the data.

Reply:
Yes sir, basic Auth.